src/main/java/com/evilmidget38/UUIDFetcher.java

```diff
@@ -92,6 +92,6 @@ public static UUID fromBytes(byte[] array) {
     }
 
     public static UUID getUUIDOf(String name) throws Exception {
-        return new UUIDFetcher(Arrays.asList(name)).call().get(name);
+        return new UUIDFetcher(Collections.singletonList(name)).call().get(name);
```

Comment:
Yep, instead of creating an entire list for no reason, the singleton list does this efficiently.